### DISCUSSION TITLE
6DOF campbell using features

### DISCUSSION THREAD

**Discussion by hserdogan94** [petrobras/ross#1098] [Q&A] [answered]
Hi @jguarato,

I checked the Issues and closed issues (e.g., #1066) to find information on the Campbell Diagram for lateral, axial, and torsional terms. I noticed that some enhancements already exist in the development version, and it seems that the "mode_type" has been added. However, I'm not entirely sure how to use this feature.

Could someone explain how to use the Campbell diagram for different types of analysis?

Thank you.

**Comment by jguarato** [comment, marked as answer]
Hi @hserdogan94,

Initially, `mode_type`  was implemented for the Campbell diagram plot of the 6 DoF model. This allows us to identify whether the mode being displayed is lateral, torsional or axial.

Please check if the discussion #1041 clarifies your questions.

**Reply by hserdogan94** [reply]
Thank you with discussion #1041 it is more clear for me.
